packages/cubejs-snowflake-driver/src/SnowflakeDriver.ts

```diff
@@ -449,7 +449,8 @@ export class SnowflakeDriver extends BaseDriver implements DriverInterface {
       );
 
       await this.execute(connection, 'ALTER SESSION SET TIMEZONE = \'UTC\'', [], false);
-      await this.execute(connection, `ALTER SESSION SET STATEMENT_TIMEOUT_IN_SECONDS = ${this.config.executionTimeout}`, [], false);
+      await this.execute(connection, 'ALTER SESSION SET STATEMENT_TIMEOUT_IN_SECONDS = ${this.config.executionTimeout}', [], false);
```

Comment:
this line change needs reverting to have the tick marks for the string interpolation

Reply:
aaaah, gotcha, sorry I assumed it needed to be consistent and the backticks were off, will fix.